org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/AbapGitView.java

```diff
@@ -506,6 +512,36 @@ public void run() {
 		this.actionPullWizard.setText(Messages.AbapGitView_context_pull);
 		this.actionPullWizard
 				.setImageDescriptor(AbstractUIPlugin.imageDescriptorFromPlugin(AbapGitUIPlugin.PLUGIN_ID, "icons/view/abapgit.png")); //$NON-NLS-1$
+
+		//Open repository in external browser
+		this.actionOpenRepo = new Action() {
+			private IRepository repo;
+
+			public void run() {
+				Object firstElement = AbapGitView.this.viewer.getStructuredSelection().getFirstElement();
+
+				if (firstElement instanceof IRepository) {
+					this.repo = ((IRepository) firstElement);
+				}
+
+				if (this.repo != null) {
+					//  Open default external browser
+					try {
+						PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser().openURL(new URL(this.repo.getUrl()));
+					} catch (PartInitException e) {
+						AbapGitUIPlugin.getDefault().getLog().log(new Status(IStatus.ERROR, AbapGitUIPlugin.PLUGIN_ID, e.getMessage(), e));
```

Comment:
Also lets display the error occurred either in eclipse status bar or in a pop-up.

Reply:
Made the changes to display the error in eclipse status bar.